uber/templates/preregistration/preregbase.html

```diff
@@ -79,7 +79,7 @@
                 title: '{% if c.PAGE_PATH in ["/preregistration/form", "/preregistration/post_form", "/preregistration/dealer_registration"] or undoing_extra %}Attending{% elif attendee %}{{ attendee.ribbon_and_or_badge }}{% endif %}',
                 description: 'Allows access to the convention for its duration.',
                 extra: 0,
-                price: {{ c.BADGE_PRICE }},
+                price: {{ badge_cost if badge_cost is defined else c.BADGE_PRICE }},
```

Comment:
This allows us to not display the attendee badge price if it's not appropriate, particularly for attendees confirming/editing a registration and attendees registering for a group.